src/main/java/edu/ucla/library/bucketeer/Job.java

```diff
@@ -70,6 +70,26 @@ public int size() {
         return myItems.size();
     }
 
+    /**
+     * Gets the failed items count of the job.
+     *
+     * @return The failed items count
+     */
+    @JsonIgnore
```

Comment:
Don't believe this is strictly necessary. Jackson only picks up on setters/getters by default. It doesn't hurt though either.